src/cdk/bootstrap.ts

```diff
@@ -304,20 +306,24 @@ export class MiraBootstrap {
             ' a --file=<stackFile> argument.')
           return
         }
-        stackFile = this.args.file
-
+        this.stackFile = this.args.file
+        if (this.stackFile.match(/.ts$/)) {
+          const T = new Transpiler(this.stackFile)
```

Comment:
what will happen if the developer will modify any file that is included by 'the file'? will it get compiled?

Reply:
as long as hte user provides a `.ts` file in the `--file` arg `tsc` will run on the entire project every time, so also the modified file will be compiled